### PR TITLE
alfalfa-client 0.2.0

### PR DESCRIPTION
**Pull request by TShapinsky**
Merge develop into main before releasing alfalfa-client

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -10,18 +10,18 @@ jobs:
...
+
+      - name: Install Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.6"
+
+      - name: Run pre-commit
+        uses: pre-commit/action@v2.0.0
+        with:
+          extra_args: --all-files
```
Comment (nllong) [inline .github/workflows/ci.yml:24]: 🧹

File: CHANGELOG.rst [hunk 2]
```diff
@@ -1,6 +1,10 @@
 Change Log
 ==========
 
+v0.2.0 (Unreleased)
+------
+* Rename graphQL endpoint from osmName to modelName. This is a breaking change.
```
Comment (nllong) [inline CHANGELOG.rst:6]: nice

File: README.md [hunk 3]
```diff
@@ -36,9 +39,18 @@ This may take some time resolving on the initial run, but subsequent runs should
...
-
 # History
-- The implemented client is previously referred to as Boptest, from the alfalfa/client/boptest.py implementation.  It has been ported as a standalone package for easier usage across projects.
+
+- The implemented client is previously referred to as Boptest, from the alfalfa/client/boptest.py implementation. It has been ported as a standalone package for easier usage across projects.
 
 # Releasing
-See [release info here](https://gist.github.com/corymosiman12/26fb682df2d36b5c9155f344eccbe404#releasing)
+
+1. Merge all branches into develop, make sure tests pass
```
Comment (nllong) [inline README.md:48]: thanks for moving this over to here! Not sure why we had a separate gist.

File: pyproject.toml [hunk 4]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "alfalfa-client"
-version = "0.1.1"
+version = "0.2.0"
 description = "A standalone client for the NREL Alfalfa application"
 authors = [
```
Comment (nllong) [inline pyproject.toml:5]: add your name to the author list?

`Tobias Shapinsky <tobias.shapinksy@nrel.gov>`

**Review by nllong — APPROVED** [review]
looks good, one comment that might result in an small action.

File: pyproject.toml (hunk 4 above)
Comment (anyaelena) [reply pyproject.toml:5]: +1!

**Pull request merged**